docs/joss/paper.md

```diff
@@ -0,0 +1,92 @@
+---
+title: 'Pyrokinetics - A python library to standardise gyrokinetic analysis'
+tags:
+  - Python
+  - gyrokinetics
+  - turbulence
+  - plasma
+  - tokamak
+  - fusion
+authors:
+  - name: Bhavin S. Patel
+    orcid: 0000-0003-0121-1187
+    affiliation: 1 # (Multiple affiliations must be quoted)
+    corresponding: true # (This is how to denote the corresponding author)
+  - name: Maurizio Giacomin
+    orcid: 0000-0003-2821-2008
+    affiliation: 2
+  - name: David Dickinson
+    orcid: 0000-0002-0868-211X
+    affiliation: 2
+  - name: Peter Hill
+    orcid: 0000-0003-3092-1858
+    affiliation: 2
+  - name: Author with no affiliation
+    affiliation: 3
+affiliations:
+ - name: Culham Centre for Fusion Energy, Abingdon OX14 3DB, UK
+   index: 1
+ - name: University of York, Heslington, Y010 5DD, UK 
+   index: 2
+ - name: Independent Researcher, Country
+   index: 3
+date: 03 August 2023
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Fusion energy offers the potential for a near limitless source of low carbon energy and is often 
+regarded as a solution for the worlds long term energy needs. To realise such a scenario requires
+the design of high performance fusion reactors capable of containing the extreme conditions
+necessary to enable fusion. Turbulence is typically the dominant source of transport in fusion
+plasmas, accounting for the majority of the particle and heat loss. Gyrokinetic modelling aims to 
+quantify the level of turbulent transport in fusion reactors and can be used to understand the 
+major drivers of turbulence. The realisation of fusion critically depends on understanding how to
+mitigate turbulent transport and thus requires high levels of confidence in the tools being
+used. Many different gyrokinetic modelling codes are available for use and pyrokinetics aims to 
+standardise the analysis of such simulations.
+
+# Statement of need
+
+[Pyrokinetics](https://github.com/pyro-kinetics/pyrokinetics) is a Python project (package: 
+[`pyrokinetics`](https://pypi.org/project/pyrokinetics/))
+that aims to simplify and standardise gyrokinetic analysis. A wide 
+variety of different gyrokinetic codes exist that utilise different input files and
+use different normalisations for plasma parameters such as densities, temperatures, velocities,
+and magnetic fields. To gain confidence in the prediction of gyrokinetic codes it is often 
+desirable to benchmark the results of one code against another. Pyrokinetics aims to ensure that
+all physical plasma parameters given as inputs to gyrokinetic codes are accurate,
+acting as an interface between different codes. Furthermore, gyrokinetics inputs can come from a
+wide variety of modelling tools outside of gyrokinetics. Pyrokinetics interfaces with
```

Comment:
Might be good to give a couple of examples of the other tools we can read inputs from?

Reply:
Do you mean like TRANSP, JETTO, GEQDSK etc.?

Reply:
I guess I meant to say from codes like TRANSP/JETTO or formats like GEQDSK?